test-project/Assets/Generated/Source/improbable/gdk/tests/ExhaustiveMapKeyTranslation.cs

```diff
@@ -359,7 +375,7 @@ internal class ComponentReplicator : ComponentReplicationHandler
             };
 
 
-            private EntityArchetypeQuery[] CommandQueries =
+            private readonly EntityArchetypeQuery[] CommandQueries =
```

Comment:
make it `const`?

Reply:
`Constants can be numbers, Boolean values, strings, or a null reference`
Can't ;)